src/app/filecoin-plus/allocators/utils/getAllocatorUrlList.ts

```diff
@@ -1,20 +1,15 @@
+import { Octokit } from '@octokit/rest'
```

Comment:
I remember you mentioned trying Octokit - I didn't get if it was necessary to keep? If so - Why do we need this?